src/sentry/tempest/tasks.py

```diff
@@ -0,0 +1,178 @@
+            return
+
+        logger.info(
+            "Fetching the latest item id failed.",
+            extra={"org_id": org_id, "project_id": project_id, "client_id": client_id},
+        )
+    except Exception:
+        logger.info(
+            "Fetching the latest item id failed.",
+            extra={"org_id": org_id, "project_id": project_id, "client_id": client_id},
+        )
+
+
+@instrumented_task(
+    name="sentry.tempest.tasks.poll_tempest",
+    queue="tempest",
+)
+def poll_tempest_crashes(credentials_id: int) -> None:
+    # FIXME: Try catch this later
+    credentials = TempestCredentials.objects.select_related("project").get(id=credentials_id)
+    project_id = credentials.project.id
+    org_id = credentials.project.organization_id  # this should work?
+    client_id = credentials.client_id
+
+    try:
+        response_text = fetch_crashes_from_tempest(
+            org_id=org_id,
+            project_id=project_id,
+            client_id=client_id,
+            client_secret=credentials.client_secret,
+            dsn=ProjectKey.objects.get_or_create(
+                use_case=UseCase.TEMPEST, project=credentials.project
+            )[
+                0
+            ].get_dsn(),  # This does generate a dsn not sure if it will work in the grand scheme of things though.
+            offset=int(
+                credentials.latest_fetched_item_id
+            ),  # Need to convert here because it is a char in the DB
+        )
+
+        try:
+            result = json.loads(response_text)
+            credentials.latest_fetched_item_id = result[
+                "latest_id"
+            ]  # TODO: Not sure why this conversion works
+            credentials.save(update_fields=["latest_fetched_item_id"])
+        except json.JSONDecodeError:
+            logger.info(
+                "Fetching the crashes failed.",
+                extra={
+                    "org_id": org_id,
+                    "project_id": project_id,
+                    "client_id": client_id,
+                    "latest_id": credentials.latest_fetched_item_id,
+                    "error": response_text,
+                },
+            )
+    except Exception:
+        logger.info(
+            "Fetching the crashes failed.",
+            extra={
+                "org_id": org_id,
+                "project_id": project_id,
+                "client_id": client_id,
+                "latest_id": credentials.latest_fetched_item_id,
+            },
+        )
+
+
+def fetch_latest_id_from_tempest(
+    org_id: int, project_id: int, client_id: str, client_secret: str
+) -> str:
+    payload = {
+        "org_id": org_id,
+        "project_id": project_id,
+        "client_id": client_id,
+        "client_secret": client_secret,
+    }
+
+    response = http.safe_urlopen(
+        url=TEMPEST_URL + "/latest-id",
+        method="POST",
+        headers={"Content-Type": "application/json"},
+        json=payload,
+    )
+    return response.text
```

Comment:
Maybe it is better if we return `response` object and then we can include more info from it in the loggers and access some other methods and properties if needed (e.g. `response.status` or `response.json()`)